changes/pr4670.yaml

```diff
@@ -0,0 +1,2 @@
+fix:
+  - "Add SendGrid to Readme and Task Library overview - [#4670](https://github.com/PrefectHQ/prefect/pull/4670)"
```

Comment:
We generally don't do changelog entries for docs updates ie this doesn't fix an actual bug users need to worry about

Reply:
Sounds good, I'll remove that file.